### PR TITLE
Snackbar notification for "Copy All Content

### LINKED ISSUE #16261: "Copy All Content" should trigger a snackbar notification
image

When using this action we should try to inform the user with a snackbar notification.

### PR DESCRIPTION
**Pull request by Soean**
## Description
Adds a snackbar notification for "Copy All Content".
Fixes #16261

copied

### CODE REVIEW

**Review by jorgefilipecosta — APPROVED** [review]
Code change looks good and it works well on my tests. Thank you for improving this feature 👍

**Comment by youknowriad** [comment]
Makes me wonder if we want to close the menu as well.

File: packages/edit-post/src/plugins/copy-content-menu-item/index.js [hunk 1]
```diff
@@ -2,17 +2,27 @@
...
 		<ClipboardButton
 			text={ editedPostContent }
 			role="menuitem"
 			className="components-menu-item__button"
-			onCopy={ () => setState( { hasCopied: true } ) }
+			onCopy={ () => {
+				setState( { hasCopied: true } );
+				createNotice(
+					'info',
+					'Content copied.',
```
Comment (mtias) [inline packages/edit-post/src/plugins/copy-content-menu-item/index.js]: Maybe "All content copied" is better?

Comment (Soean) [reply packages/edit-post/src/plugins/copy-content-menu-item/index.js]: Thanks, thats better. I updated the code.

**Comment by Soean** [comment]
> Makes me wonder if we want to close the menu as well. There might be some a11y/design issues if we decide to close the menu, so worth checking.

I think it's a good idea. I added the label to get feedback from the A11y team.

**Comment by mtias** [comment]
Looks good to me. I think the question about whether the menu should close after the interaction can be asked as a new issue, too.

**Review by mtias — APPROVED** [review]

**Comment by Soean** [comment]
Thanks, I opened #16279 for this enhancement. When the tests pass, we can merge this PR.

**Pull request merged**